aws/aws-monitoring-role-create-tform.tf

```diff
@@ -0,0 +1,45 @@
+locals {
+  ternary_external_id = "<INSERT_EXTERNAL_ID>"
```

Comment:
this could be a [Terraform Variable](https://developer.hashicorp.com/terraform/language/values/variables) instead, if you want to compel the user to input it.

Reply:
I can update it- do we want to retro fit this to the Service Agent role as well?

Reply:
Eventually yes, but it can happen outside of this change, if you're trying to get this out the door.

Reply:
Works for me!